example/gardener-local/gardenlet/values-ipv6.yaml

```diff
@@ -0,0 +1,38 @@
+config:
+  featureGates:
+    IPv6SingleStack: true
+  gardenClientConnection:
+    bootstrapKubeconfig:
+      kubeconfig: |
+        apiVersion: v1
+        kind: Config
+        current-context: gardenlet-bootstrap
+        clusters:
+        - cluster:
+            insecure-skip-tls-verify: true
+            server: https://kubernetes.default.svc:443 
+          name: default
+        contexts:
+        - context:
+            cluster: default
+            user: gardenlet-bootstrap
+          name: gardenlet-bootstrap
+        users:
+        - name: gardenlet-bootstrap
+          user:
+            token: 07401b.f395accd246ae52d
+  seedConfig:
+    spec:
+      networks:
+        ipFamilies:
+        - IPv6
+        nodes: fc00:f853:ccd:e793::/64
+        # Those CIDRs must match those specified in the kind Cluster configuration.
+        pods: fd00:10:1::/56
+        services: fd00:10:2::/112
+        shootDefaults:
+          ipFamilies:
+          - IPv6
+          pods: fd00:10:3::/56
+          services: fd00:10:4::/112
```

Comment:
The `nodes` CIDR is determined by the range of the `kind` docker network. This is configured by kind and we don't want to make it more complicated by requiring devs to recreate the `kind` network with a "readable" range.

Reply:
Would it not be possible to automate the creation of the `kind` network and still achieve nice IP addresses?
I completely agree that additional manual steps should be omitted wherever possible, but the `kind` cluster is created by "us". Therefore, we should be able to configure it properly.
WDYT?

Reply:
Fair enough. We also tell developers to recreate the kind network if it gets created with the "wrong" IPv4 CIDR (ref https://github.com/gardener/gardener/pull/6935). This could be automated as well.
I will give it a try and add a new commit to this PR to tackle both problems.

Reply:
Done in a23065d9514f69f2b42f3bc0ef859fd05ffa05d8, let me now what you think :)